dashboard/test/ui/features/learning_platform/projects/applab_project.feature

```diff
@@ -103,8 +103,6 @@ Scenario: Save Project After Signing Out
   When I sign out using jquery
   And I add code "// comment 2" to ace editor
   And ace editor code is equal to "// comment 1// comment 2"
-  And I press "resetButton"
```

Comment:
do we expect line 105 to redirect the user? in [the log](https://cucumber-logs.s3.amazonaws.com/test/test/IE11_learning_platform_projects_applab_project_output.html?versionId=dSU_F2nSysPw9qKTMeuw0.Os.D9aPLED) it looks like clicking resetButton is what prompted the browser to redirect, but that may not be what's actually happening

Reply:
When I did it by hand, I didn't click the reset button - I saw a message that there was an error saving the project (where the save timestamp would usually be) and then it redirected. I can dig into this a little more though

Reply:
no, that's okay -- the log is probably just misleading because the redirect doesn't happen immediately after project save fails

Reply:
I think what actually happened is our test driver completes actions much faster than a human. It was able to add text and click the reset button before the page finished the redirect action. It seems like this test was initially intended to test a different functionality (page redirect when the run button is clicked) but when the code was updated to redirect on signout, the test continued due to a race condition that the test driver always won.

Reply:
I take that back. I didn't realize the test wasn't just hitting the "sign out" button. It's hitting the sign out endpoint.

Reply:
I take that back. I didn't realize what "sign out with jquery" does. (no automatic page reload)

Reply:
If this ends up not working, another option is to disable the test temporarily on IE.